src/app.js

```diff
@@ -1 +1,35 @@
 'use strict';
+const readline = require('readline');
+const getRandomNum = require('./getRandomNum');
+const calculateResult = require('./calculateResult');
+
+const terminal = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const randomNumber = String(getRandomNum());
+
+const game = () => {
+  terminal.question('Write your number of four digits: ', (inputNumber) => {
```

Comment:
Add input validation to ensure the user entered a 4-digit number with unique digits.